docs/traffic-policy/examples/index.mdx

```diff
@@ -32,14 +32,38 @@ Explore a curated collection of examples and configuration examples spanning fro
 
 A number of these examples come from a longer article about how ngrok [makes policy management accessible](https://ngrok.com/blog-post/api-gateway-policy-management-examples) to developers, including a simple Go-based application for testing these and other configurations.
 
-See the following categories for specific expressions and actions:
+## Route requests
 
-- [Authentication](#authentication)
-- [Rate limiting](#rate-limiting)
-- [Block unwanted requests](#block-unwanted-requests)
-- [Other](#other)
+### Forward to an internal endpoints based on request attributes
 
-## Authentication
+You can use [CEL
+interpolation](/docs/traffic-policy/concepts/cel-interpolation.mdx) to
+dynamically forward requests to different internal endpoints based on URL,
+hostname, path, headers, and more. This allows you to manage complex traffic
+topologies without manually creating and managing each [`forward-internal`
+action](/docs/traffic-policy/actions/forward-internal.mdx).
+
+Forward requests from `https://*.example.com` to `https://<SUBDOMAIN>.internal`.
+
+```yaml mode=traffic-policy
+on_http_request:
+  - actions:
+      - type: forward-internal
+        config:
+          url: https://${req.host.split('.example.com)[0]}.internal
```

Comment:
Would this actually work, considering the single apostrophe at `'.example`?
`req.host.split('.example.com)[0]`